src/YaccConstructor/Program.fs

```diff
@@ -44,6 +44,32 @@ let log (e:System.Exception) msg =
     "\nMessage:\n  " + msg
     |> printfn "%s"
 
+open Argu
+
+type CLIArguments =
+    | [<Inherit; AltCommandLine("-f","-frontend")>] F of feName:string option
```

Comment:
Please use more descriptive constructor names. It's more difficult to get what they are in other places in the code.